metron-platform/metron-solr/src/main/java/org/apache/metron/solr/dao/SolrMetaAlertRetrieveLatestDao.java

```diff
@@ -36,8 +37,10 @@
     MetaAlertRetrieveLatestDao {
 
   private SolrDao solrDao;
+  private SolrClient solrClient;
 
-  public SolrMetaAlertRetrieveLatestDao(SolrDao solrDao) {
+  public SolrMetaAlertRetrieveLatestDao(SolrClient client, SolrDao solrDao) {
```

Comment:
I like this approach of constructor dep injection for all DAOs. Independent of a larger refactoring effort, I think we should standardize on at least passing in a solrclient via constructor for all DAOs. We really shouldn't be passing around connection/client objects like stolen candy between them. If we're decorating/composing objects, then let's wire this up via constructors.